controlplane/src/bin/delete-user.ts

```diff
@@ -24,10 +27,17 @@ const {
   databaseTlsKey,
   webhookUrl,
   webhookSecret,
+  s3Storage,
 } = getConfig();
 
 const userId = process.env.USER_ID || '';
 
+const bucketName = extractS3BucketName(s3Storage);
```

Comment:
Is it really necessary to put all the logic in here? Why don't we issue a job to the queue and the controlplane picks it up? In that way, we also leverage all benefits that the queue provides retry, history etc...

Reply:
This is run in GH actions which triggers a k8 job and it is transactional. It immediately gives us why a user can/cannot be deleted and what goes wrong. The benefits you listed is availbe with actions itself